partcad/src/partcad/part_factory_scad.py

```
@@ -86,3 +89,108 @@ async def instantiate(self, part):
             self.ctx.stats_parts_instantiated += 1
 
             return shape
+
+    def pre_process_scad(self):
```

Ruff (0.8.2)

`pre_process_scad` is too complex (15 > 10)

(C901)